docs: users should land on new tracker by default

teleivo opened the pull request:
its been at least 2 years since new tracker has been released (from 2.36 according to our docs). New tracker is far from new anymore. We've added a disclaimer to the old tracker docs some time ago that anyone starting out should use the new tracker APIs. Its time to also rearrange the pages so new users land on the new tracker APIs by default.

This is how the pages should show up in the sidebar on the left

* Tracker
* Tracker (Old)

I've used the word old as its the one we are using internally and its the antonym for new. Let me know if you another name makes more sense 👌🏻

enricocolasante commented on src/developer/web-api/tracker.md:
```diff
@@ -1,531 +1,1357 @@
->  feature you will have to postpone the migration until a new SYNC feature is
->  in place.
+> - The old endpoints are marked as deprecated but still work as before.
+> - Some functionality is not yet ready in the new endpoints, but they support their primary use-cases.
```
I would rephrase that and maybe specify that just sync is not working in the new endpoints but everything else is ready

teleivo committed:
docs: users should land on new tracker by default

its been at least 2 years since new tracker has been released (from 2.36
according to our docs). New tracker is far from new anymore. We've added
a disclaimer to the old tracker docs some time ago that anyone starting
out should use the new tracker APIs. Its time to also
rearrange the pages so new users land on the new tracker APIs by
default.
